### DISCUSSION TITLE
Compile error due to readr

### DISCUSSION THREAD

**Discussion by HWelinkim** [jtr13/cc21fall1#87] [Q&A]
This is for PR #30 . 

The following line of code is erroring out: 

```{r }
# read in data
df <- readr::read_csv("https://storage.googleapis.com/kagglesdsdata/datasets/9961/14084/Train.csv?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gcp-kaggle-com%40kaggle-161607.iam.gserviceaccount.com%2F20211102%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20211102T150038Z&X-Goog-Expires=259199&X-Goog-SignedHeaders=host&X-Goog-Signature=22b981f7d24fe5629802b5c5556955d7609a9bd1a5d12171320b9cc76662012a43e69dbd6d62fd199765e8168948168d7c793bd7f6006efd1b51dfc816649b80f24dc88d5e70f20a60cf7f6a5580093b62f7fb7648c532b791f97019fc2a726fb543de575ba7811afa656c2c59a3516dc90d4afc601292d5fc7125dca38d612d928f6766f1ea89d17e0ee261cd83fb4aa2fff24aa796db172b2a90c4e9b8cac6619f56f831e016221a028810a4442a69bc393c46a3a25b4ac9e6bbf89f80c1c01a4b873bd484bc7c7ea06d478985495c762eaea3226a2e0968a6e9ccf7ae22db64d8def323425dc785f2478d4c0e05322275cabe3f987b60fdd3f99d08f1eec6"
```
The error message is as follows: Error in open.connection(4L, "rb") : HTTP error 400.

Could you please help investigate this issue? @jtr13 @s10singh97 @Shruti-Kaushal

**Comment by jtr13** [comment]
Maybe it needs an API key, I'm not sure. Please make a note on the PR that it's not working and ask the authors to submit a new PR to fix it by providing a working link.

**Comment by s10singh97** [comment]
> This is for PR #30 .
> 
> The following line of code is erroring out:
> 
> ```
> # read in data
> df <- readr::read_csv("https://storage.googleapis.com/kagglesdsdata/datasets/9961/14084/Train.csv?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gcp-kaggle-com%40kaggle-161607.iam.gserviceaccount.com%2F20211102%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20211102T150038Z&X-Goog-Expires=259199&X-Goog-SignedHeaders=host&X-Goog-Signature=22b981f7d24fe5629802b5c5556955d7609a9bd1a5d12171320b9cc76662012a43e69dbd6d62fd199765e8168948168d7c793bd7f6006efd1b51dfc816649b80f24dc88d5e70f20a60cf7f6a5580093b62f7fb7648c532b791f97019fc2a726fb543de575ba7811afa656c2c59a3516dc90d4afc601292d5fc7125dca38d612d928f6766f1ea89d17e0ee261cd83fb4aa2fff24aa796db172b2a90c4e9b8cac6619f56f831e016221a028810a4442a69bc393c46a3a25b4ac9e6bbf89f80c1c01a4b873bd484bc7c7ea06d478985495c762eaea3226a2e0968a6e9ccf7ae22db64d8def323425dc785f2478d4c0e05322275cabe3f987b60fdd3f99d08f1eec6"
> ```
> 
> The error message is as follows: Error in open.connection(4L, "rb") : HTTP error 400.
> 
> Could you please help investigate this issue? @jtr13 @s10singh97 @Shruti-Kaushal

The storage api token has expired and therefore it is not able to extract the file.
Added eval = FALSE to all their r chunks and uncommented the file from `_bookdown.yml` file.

**Reply by HWelinkim** [reply]
@s10singh97 Just wanted to confirm that there is no further action needed for this PR. Please let me know if any other actions are needed.

**Reply by s10singh97** [reply]
Apologies for the delay. Yes, no further action is needed.
Great Work!
